.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+node_js:
+  - "stable"
+cache:
+  directories:
+    - node_modules
+
+install:
+  - npm install
+  - npm run client:init
+
+script:
+  - npm test
+  - npm run test:client
+  - npm run test:coverage
+  - npm run test:client:coverage
+  - npm run client:build
+
```

Comment:
Sorry, my mistake - you don't need to run the build script.

You only need your tests with coverage here, you can leave out the others
```
script:
  - npm run test:coverage && npm run test:client:coverage
```